posthog/models/sessions/sql.py

```diff
@@ -73,6 +74,14 @@
     engine=SESSIONS_DATA_TABLE_ENGINE(),
 )
 
+
+def source_column(column_name: str) -> str:
+    if EE_AVAILABLE:
+        return f"mat_{column_name}"
+    else:
+        return f"JSONExtractString(properties, '{column_name}')"
```

Comment:
nice 💯 

there's https://github.com/PostHog/posthog/blob/8b7b97ef09f9bef606dceaa0af360305d8212f69/posthog/models/property/util.py#L714 that we could re-use which should let this use materialized columns more safely but regardless this is a great improvement.

Reply:
(and even then that would need amending to use jsonextractstring conditionally anyway so 6 of one etc etc)